aemedge/blocks/hero/hero.js

```diff
@@ -12,19 +16,38 @@ function calculateInitials(name) {
   return initials;
 }
 
-function decorateMetaInfo() {
-  const infoBlockWrapper = div({ class: 'media-blend__info-block' });
+function buildAuthorEl(author) {
+  const authorUrl = `/author/${toClassName(author.trim()).replaceAll('-', '')}`;
```

Comment:
The generation of the author URL could be move to utils.js as well as we need that on other place. Maybe check with @saurabh-khare  & @pahupe as they work on #312

Reply:
Have created a new util function.

Reply:
@rjwtrmn : Please also see https://adobe-dx-support.slack.com/archives/C06RDNH0C95/p1711641332448679